src/actions/sso/index.ts

```diff
@@ -1,26 +1,47 @@
 import { Dispatch, AnyAction } from 'redux'
-import { BackendMiddleware } from '../../backendMiddleware'
 import { CredentialRequest } from 'jolocom-lib/js/credentialRequest/'
 import { VerifiableCredential } from 'jolocom-lib/js/credentials/verifiableCredential'
+import { StateCredentialRequestSummary, StateAttributeSummary, StateTypeSummary, StateVerificationSummary } from 'src/reducers/sso'
+import { BackendMiddleware } from 'src/backendMiddleware'
+import { navigationActions } from 'src/actions'
+import { routeList } from 'src/routeList'
+import { showErrorScreen } from 'src/actions/generic'
+
+export const setCredentialRequest = (request: StateCredentialRequestSummary) => {
+  return {
+    type: 'SET_CREDENTIAL_REQUEST',
+    value: request
+  }
+}
+
+export const clearCredentialRequest = () => {
+  return {
+    type: 'CLEAR_CREDENTIAL_REQUEST'
+  }
+}
 
 export const consumeCredentialRequest = (jwtEncodedCR: string) => {
   return async(dispatch: Dispatch<AnyAction>, getState: Function, backendMiddleware: BackendMiddleware) => {
     const { storageLib } = backendMiddleware
     const CR = new CredentialRequest().fromJWT(jwtEncodedCR)
```

Comment:
I would rename it to `credentialRequest`, `CR` is confusing - a name of the const should not be capitalized, and we shouldn't have to think about how to extend the shortcut while reading it.

Reply:
Agree, also can be confused with credentialResponse, which would also abbreviate to CR :(